### ISSUE TITLE
waitforpopup not compiling

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#14]
```
Operating system :
.Net Framework version : 4
Office Version : Excel 2007
SeleniumWrapper version : 1.0.8.0

What steps will reproduce the problem?
code used is;
dim rc as integer
Dim selenium As New SeleniumWrapper.WebDriver

rc = selenium.waitForPopUp("Booking", "300")

To wait for a popup called 'booking'

What is the expected output? What do you see instead?

Expected is for the code to run, but I get 'Expected Function or variable' 
instead. Because Excel won't allow the code to run I don't know whether the 
function itself is ok or not.


Please provide any additional information below.

```

Original issue reported on code.google.com by `slimydr...@gmail.com` on 11 Feb 2013 at 12:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `florentbr` on 13 Feb 2013 at 4:44
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
waitForPopUp is not compiling because it has no return value.
The correct syntax is :
selenium.waitForPopUp "Booking", "3000"

Anyway, while trying to reproduce your issue, i found another on in the .Net 
wrapper :
http://code.google.com/p/selenium/issues/detail?id=5160

This function won't work as long as it's not fixed in the .Net wrapper which is 
used by this VBA wrapper.

```

Original comment by `florentbr` on 13 Feb 2013 at 11:20

**Comment by GoogleCodeExporter** [comment]
```
Temporarily fixed in version 1.0.10
```

Original comment by `florentbr` on 20 Feb 2013 at 12:11

**Comment by GoogleCodeExporter** [comment]
```
Fully fixed in version 1.0.11
```

Original comment by `florentbr` on 5 Mar 2013 at 12:51

**Comment by GoogleCodeExporter** [comment]
Original comment by `florentbr` on 5 Mar 2013 at 12:52
- Changed state: **Fixed**
